docs/2.guide/2.directory-structure/1.server.md

```diff
@@ -255,6 +255,16 @@ export default defineEventHandler((event) => {
 })
 ```
 
+### Sending Redirect
+
+```ts [server/api/foo.get.ts]
+export default defineEventHandler(async (event) => {
+  return sendRedirect(event, '/path/redirect/to', 302)
+  // or
+  // await sendRedirect(event, '/path/redirect/to', 302)
```

Comment:
awaiting only is not preferred. We should always return to ensure control flow terminates at this point.

Reply:
Good catch 👍🏻